tests/manual-test-cases/Group5-Functional-Tests/5-29-Opaque-Network.md

```diff
@@ -9,17 +9,25 @@ To verify the VIC appliance works with NSXT Logical Switches
 
 # Environment:
 This test requires to use VDNET 3.0 for vCenter, ESXi and NSX-T creation and customization
-
-# Test Steps:
 1. Deploy a new topology which includes vCenter and NSXT in Nimbus using VDNET 3.0
 ```A new vCenter```
 ```A new NSXT Manager, NSXT Controller and NSXT Edge```
 ```Three ESXi Hosts and put two of them into os-computer-cluster-1 cluster and one of them into management cluster```
 ```Create an overlay transport zone in NSXT```
-2. Create two overlay logical switches for bridge and container network
-3. Install VIC appliance and deploy a VCH
-4. Run a variety of docker commands on the VCH appliance
-5. Destroy testbeds
+
+# Case 1 Test Steps:
+1. Create two overlay logical switches for bridge and container network
+2. Install VIC appliance and deploy a VCH
+3. Run a variety of docker commands on the VCH appliance
+4. Destroy the created VIC appliance and VCH
+
+# Case 2 Test Steps:
+1. Create a new logical switches for bridge and container network
```

Comment:
2 new logical switches?

Reply:
Yes, I will create 2 VCH and each one uses different LS as bridge network.